src/BlazorUI/Bit.BlazorUI/Components/Navs/Pagination/BitPagination.scss

```
@@ -47,32 +63,167 @@
+    font-size: inherit;
+    align-items: center;
+    white-space: nowrap;
+    gap: spacing(0.5);
+    padding: 0 spacing(0.5);
+    height: var(--bit-pgn-btn-size);
+}
+
+.bit-pgn-gti,
+.bit-pgn-pse {
+    font: inherit;
+    // The height is the one every control of the pagination stands at, so the border is counted into it
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)